### DISCUSSION TITLE
Problem when selecting contact photos

### DISCUSSION THREAD

**Discussion by desk7** [FossifyOrg/General-Discussion#680] [Q&A] [answered]
I have installed fossify contacts on a phone with android 14.

When I try to add a photo to an existing contact, I select a photo from the gallery, then the app shows the option to trim the image. The problem is that the button `save a copy` is greyed out unless I trim a bit the image.
Moreover, for a particular photo, after rotating the image, the button 'automatic' appears. if I click on it, the image turns back to the original orientation but now the button 'save a copy' is clickable.

Is this a bug or are there some problems with the images?

**Comment by Aga-C** [comment, marked as answer]
Fossify Contacts doesn't have its own editor for trimming photos. It uses a default one set on your phone. If you can't save the image, it's not a fault of Fossify Contacts, but your photo editor. Everything works fine with the editor from Fossify Gallery.

**Reply by desk7** [reply]
You're right! The problem was the Google photo editor. I installed the Fossify Gallery and all works well.
